testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AbstractIntegrationSpec.groovy

```diff
@@ -548,8 +547,9 @@ tmpdir is currently ${System.getProperty("java.io.tmpdir")}""")
         failure.assertThatCause(containsNormalizedString(description))
     }
 
+    // TODO: Inline and remove.
     protected Matcher<String> containsNormalizedString(String description) {
```

Comment:
What's this about?

Reply:
I extracted the method to `Matchers` in order to move a Javadoc options output test to a unit test. I think that this method should be inlined so we don't have this unnecessary redirection, but it would change a lot of tests and I wanted to avoid the noise in this PR.